samtranslator/model/connector/connector.py

```diff
@@ -20,6 +20,11 @@
     ],
 )
 
+STANDARD_CONNECTOR = "StandardConnector"
+EMBEDDED_CONNECTOR = "EmbeddedConnector"
+SOURCE_CONNECTOR = "SourceConnector"
+EMBEDDED_SOURCE_CONNECTOR = "EmbeddedSourceConnector"
```

Comment:
Should we add this now or when we actually implement it?

Reply:
I'm not sure if I'm following. These are being used now.

Reply:
It's already supported after we implement `EMBEDDED_CONNECTOR` and `SOURCE_CONNECTOR`

Reply:
Ah I didn't realize `EMBEDDED_SOURCE_CONNECTOR` was multi dest in embedded connector. Much cleared with the name change